### PR TITLE
UI Improvements

### LINKED ISSUE #3: unified_inventory does not support private skins
The UE integration differs to the sfinv or smart_inventory integration because the inventory is stateless and does usually pre-generate page formspecs by design. Therefore the UE does not support the private skins because different per-player pages required.

Some points already discussed in https://github.com/minetest-mods/skinsdb/issues/1

### PR DESCRIPTION
**Pull request by bell07**
Reopens #6 because of git command line trouble.
Please review (anyone) before I merge them

Includes previous PR #5 for issue #2
formspec parts consolidated to formspec.lua removing redundancy
unified_inventory supports private skins now, fixes #3
new chat command /skinsdb show to show current skin info in a formspec
new chat command /skinsdb show to show skin info in a formspec
new chat command /skinsdb ui (or just /skinsdb) to show select skin in formspec without inventory

### CODE REVIEW

File: chatcommands.lua [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		-- parse command line
+		local command, parameter
+		for word in param:gmatch("([^ ]+)") do
+			if not command then
+				-- first word
+				if word == 'set' or word == 'list' or word == 'show' or word == 'ui' then
+					command = word
+				elseif skins.get(word) then
+					command = 'set'
+					parameter = word
```
Comment (SmallJoker) [inline chatcommands.lua]: Using `string.split` might be more efficient here.

File: chatcommands.lua [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+				list = skins.get_skinlist_for_player()
+			elseif not parameter then
+				list = skins.get_skinlist_for_player(name)
+			else
+				return false, "unknown parameter", parameter
+			end
+
+			local current_skin_key = skins.get_player_skin(player):get_key()
+			for _, skin in ipairs(list) do
+				local info = skin:get_key()..": name="..skin:get_meta_string("name").." author="..skin:get_meta_string("author").." license="..skin:get_meta_string("license")
```
Comment (SmallJoker) [inline chatcommands.lua]: A bit long lines are fine but this one should really be split into two.

**Comment by bell07** [comment]
Thanks for review. Both solved. In addition added intllib support to the chat commands in the same way as done in formspecs. Translated to german

**Comment by bell07** [comment]
Is this PR approved to be merged?

**Comment by sofar** [comment]
There's a lot mixed up in this PR. I wouldn't merge a PR that looks like this. Do you need help cleaning it up and rebasing it?

**Comment by bell07** [comment]
I did a rebase to the current master. See 3 enhancements: chatcommands, UI consolidation and Malay/German translations, but all aofthem are depending on each other, so I do not see the reason or the right way how to split them. Please help me if you still see something to do.

**Comment by sofar** [comment]
This looks better already. @SmallJoker let's just merge this?

**Pull request merged**